sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/MapElementsTest.java

```diff
@@ -148,7 +148,7 @@ public Integer apply(Integer input) {
 
   @Test
   public void testSimpleFunctionDisplayData() {
```

Comment:
Added these tests, but moved the commit to a different branch that is not yet in a PR. Once #757 is merged, it will unblock this PR as well as enable opening a PR to add display data to `SimpleFunction`, which is no longer on the critical path for me.